.github/workflows/bridge-ui-v2.yml

```diff
@@ -24,6 +24,13 @@ jobs:
       - name: Install pnpm dependencies
         uses: ./.github/actions/install-pnpm-dependencies
 
+      # This step is needed because the .env file is not committed to the repository
+      # and the Svelte check complains about not finding the exports from $env/static/public,
+      # which will be generated based on the .env file when running `svelte-kit sync`
+      - name: Make dummy envfile
```

Comment:
nit: rename to to "Copy .env.example to .env"

Reply:
Done